core/src/style/textStyle.cpp

```diff
@@ -175,10 +174,9 @@ void TextStyle::onBeginDrawFrame(const View& _view, const Scene& _scene) {
         m_shaderProgram->setUniformMatrix4f("u_proj", glm::value_ptr(_view.getOrthoViewportMatrix()));
         m_dirtyViewport = false;
     }
+    
+    Style::onBeginDrawFrame(_view, _scene);
```

Comment:
All styles have a default "empty" material, so calling a setup function on it is slightly wasteful, but not harmful. `TextStyle` and its brethren don't *yet* define `u_zoom`, but they will eventually because it must be made available for custom shader blocks.